pandas/core/series.py

```diff
@@ -494,7 +495,7 @@ def __init__(
             if not is_list_like(data):
                 data = [data]
             index = default_index(len(data))
-        elif is_list_like(data):
+        elif is_list_like(data) and not isinstance(dtype, ListDtype):
```

Comment:
What about nested list?

Reply:
Yea this is a tough one to handle. I'm not sure if something like:

```python
pd.Series([1, 2, 3], index=range(3), dtype=pd.ListDtype())
```

should raise or broadcast. I think the tests currently want it to broadcast, but we could override that expectation for this array

Reply:
good example.  also indexing is going to be a pain point.  a lot of checking for nested listlikes happens in indexing.py and im wary of stumbling blocks there.  also indexing in general with a ListDtype index.  i think the solution is to interpret lists as sequences and only treat them as a scalar if explicitly wrapped in pa.ListScalar (or something equivalent)

Reply:
I originally started down that path but I _think_ the problem with expecting `pa.ListScalar` is that when you select from a ListArray, I think most users would expect a plain python list. So to use that same selection as an indexer I think its hard to avoid the built-in type

Reply:
> So to use that same selection as an indexer

I think we have to just disallow that.  Otherwise we have a long tail of places where we expect labels to be non-listlike and nested objects to mean 2D.  e.g. `df.set_index(list_column).T.set_index([1, 2, 3])` is ambiguous whether it wants three columns to be set as an index or a single column with label `[1, 2, 3]`.  I'm particularly worried about indexing.py code where we check for nested data getting way more complicated if the behavior has to depend on dtypes.

Reply:
> `df.set_index(list_column).T.set_index([1, 2, 3])` is ambiguous whether it wants three columns to be set as an index or a single column with label `[1, 2, 3]`

Do we normally require index elements to be hashable? I think there's an argument to be made that we shouldn't allow this as a index, given Python lists aren't hashable (even though PyArrow lists are)

If we do want to allow this as an index, the example cited is a single column with label [1, 2, 3]

> I'm particularly worried about indexing.py code where we check for nested data getting way more complicated if the behavior has to depend on dtypes.

Definitely a valid concern. Do you know what might not be covered in this area by the extension tests?

Reply:
Yea that's a good point about the groupby stuff. I don't have a strong objection to requiring an explicit scalar argument, though I guess the follow up question is do we want users passing a pyarrow scalar or do we want to create our own scalar type.

Not sure how other libraries like polars handle this, but @MarcoGorelli might have some insight